src/Components/ArticleCard.jsx

```diff
@@ -0,0 +1,22 @@
+import Card from 'react-bootstrap/Card'
+import {useParams} from 'react-router'
```

Comment:
you've got `useParams` being imported but it's unused in this component so it could be cleaned up by removing the line 🧹 ✨